src/sample/index.ts

```diff
@@ -18,15 +19,21 @@ export class ExampleStack extends cdk.Stack {
 
     // user's lambda function
     const hello = new lambda.Function(this, "HelloHandler", {
-      runtime: lambda.Runtime.NODEJS_10_X,
+      runtime: lambda.Runtime.NODEJS_14_X,
```

Comment:
Good that we're switching this from Node 10 to Node 14, since Node 10 is no longer supported. However, I still see some other uses of Node 10 in this file, can we switch those as well?